src/lib/parsers.ts

```diff
@@ -29,33 +29,32 @@ import {
 
 /**
  * Different events use different names for teams:
- * 
+ *
  * - `blessMaiden` uses `"Red"` and `"Blue"`
  * - `victory` uses `"Gold"` and `"Blue"`
- * 
+ *
  * This object covers all the possibilities and maps
  * them to a single enum.
  */
```

Comment:
Move this comment to `KQStream#normalizeValues` to explain why we rename "Red" to "Gold".